jenkins/ci.suse.de/cloud-maintenance.yaml

```diff
@@ -0,0 +1,116 @@
+
+- project:
+    name: cloud-ardana-job-mu-entry-scale-kvm-x86_64
+    reserve_env: false
+    updates_test_enabled: false
+    scenario_name: entry-scale-kvm
+    clm_model: standalone
+    controllers: '3'
+    computes: '2'
+    ses_enabled: true
+    ses_rgw_enabled: false
+    extra_params: |
+      tempest_retry_failed=True
+      update_services_serial=True
+    triggers: []
+    ardana_job:
+      - cloud-ardana8-job-mu-entry-scale-kvm-deploy-x86_64:
+          cloudsource: GM8+up
+          update_after_deploy: false
+          tempest_filter_list: "\
+            keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,vpnaas,lbaas,\
+            designate,heat,ceilometer,magnum,freezer,monasca"
+      - cloud-ardana8-job-mu-entry-scale-kvm-update-x86_64:
+          cloudsource: GM8+up
+          update_after_deploy: true
+          tempest_filter_list: "\
+            keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,vpnaas,lbaas,\
+            designate,heat,ceilometer,magnum,freezer,monasca"
+      - cloud-ardana9-job-mu-entry-scale-kvm-deploy-x86_64:
+          cloudsource: GM9+up
+          update_after_deploy: false
+          tempest_filter_list: "\
+            keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,lbaas,\
+            designate,heat,magnum,monasca"
+      - cloud-ardana9-job-mu-entry-scale-kvm-update-x86_64:
+          cloudsource: GM9+up
+          update_after_deploy: true
+          tempest_filter_list: "\
+            keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,lbaas,\
+            designate,heat,magnum,monasca"
+    jobs:
+        - '{ardana_job}'
+
+
+- project:
+    name: cloud-crowbar-job-mu-no-ha-x86_64
+    reserve_env: false
+    updates_test_enabled: false
+    scenario_name: standard
+    controllers: '1'
+    computes: '1'
```

Comment:
On RC You mentioned possibility that we could use 2 computes instead of one, at least for tempest and I agree that it makes sense to use 2 computes even for non-ha.

Reply:
done